nominees/sempo.json

```diff
@@ -10,38 +13,51 @@
   ],
   "SDGs": [
     {
-      "SDGNumber": 1
+      "SDGNumber": 1,
+      "evidenceText": "Unblocked Cash, powered by Sempo, is a blockchain-based Cash and Voucher Assistance (CVA) solution that enables faster, less expensive, and more transparent financial aid for relief efforts. Along with our partners we are changing the way that humanitarian organisations respond to disasters. ",
+      "evidenceURL": "https://www.youtube.com/watch?v=kOA_9THMC2c&t=1s"
```

Comment:
Assuming Sempo is aiming for Target 1.4 of SDG 1, are they able to provide evidence of Sempo being used in this specific context, particularly in acquiring secure tenure rights to land?

Generally, though, it does seem they meet this SDG by what I can see reviewing their product website…

![Screenshot of Target 1.4 of SDG 1: "By 2030, ensure that all men and women, in particular the poor and the vulnerable, have equal rights to economic resources, as well as access to basic services, ownership and control over land and other forms of property, inheritance, natural resources, appropriate new technology and financial services, including microfinance."](https://user-images.githubusercontent.com/4721034/154732964-dbaf744b-e964-47d2-b87d-7bf48f7574df.png "Screenshot of Target 1.4 of SDG 1: \"By 2030, ensure that all men and women, in particular the poor and the vulnerable, have equal rights to economic resources, as well as access to basic services, ownership and control over land and other forms of property, inheritance, natural resources, appropriate new technology and financial services, including microfinance.\"")